src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-	  <VersionPrefix>16.0.0</VersionPrefix>
+	  <VersionPrefix>16.0.1</VersionPrefix>
 	  <!--For Pre-release packages update the VersionSuffix -->
-	  <VersionSuffix></VersionSuffix>
+	  <VersionSuffix>alpha1</VersionSuffix>

```

Comment:
will go ahead and remove this since we won't need to push this up to nuget to test